packages/tools/monorepo-test-orchestrator/src/index.ts

```
@@ -0,0 +1,591 @@
+    
+    let shards;
+    switch (this.config.shardStrategy) {
+      case 'dependencies':
+        shards = this.workDivider.divideByDependencies(workItems, dependencyGraph);
+        break;
+      case 'size':
+        shards = this.workDivider.divideBySize(workItems, optimalWorkers);
+        break;
+      case 'complexity':
+      default:
+        const threshold = workItems.reduce((sum, item) => sum + item.complexity, 0) / optimalWorkers;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)